decidim-elections/spec/cells/decidim/elections/election_cell_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::Elections::ElectionCell, type: :cell do
+  controller Decidim::Elections::ElectionsController
+
+  subject { my_cell.call }
+
+  let(:my_cell) { cell("decidim/elections/election", model) }
+  let!(:election) { create(:election, :complete, :published, :ongoing) }
+  let!(:current_user) { create(:user, :confirmed, organization: model.participatory_space.organization) }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(current_user)
+  end
+
+  context "when rendering an election" do
+    let(:model) { election }
+
+    it "renders the card" do
+      expect(subject).to have_css(".card--election")
```

Comment:
can we also check that mandatory fields are rendered, like title, ...?

Reply:
Hi @tramuntanal, thanks for the review! The image is removed already. At least outside `design-app`. And sure, I'll add some more expectations and update the PR.👍

Reply:
I checked locally and you are right! I don't know why GitHub doesn't tells me that the file was deleted:
![imatge](https://user-images.githubusercontent.com/199462/91557070-12ba6c80-e934-11ea-876b-07f1ac283702.png)

Reply:
I don't know. 😊 I added some more tests for the `cell` and removed the commits that were already merged, so we have a "clean" PR. With that, I had to force push my changes. (_Just saying because you merged `develop` into this PR._)